sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java

```diff
@@ -119,6 +120,10 @@ protected void addFields() {
 		epidField.setInvalidCommitted(true);
 		style(epidField, ERROR_COLOR_PRIMARY);
 
+		TextField externalIdField = addField(CaseDataDto.EXTERNAL_ID, TextField.class);
+		externalIdField.setInvalidCommitted(true);
```

Comment:
Not necessary, the external ID doesn't have any warning associated with it that would prevent the form from being saved

Reply:
Then should I remove it from epidField as well (see above)?

Reply:
No, the epid number field has this warning associated